UKHomeOffice/sgmr-service
# SPCR-419 Handle people with expired documents

### AC / Description of changes
If a person has an expired document they should not be allowed to be submitted in a voyage report

### To Test
- for the purposes of testing change `today`'s value in `date.js` file to a future date that you know is later than at least one of your saved peoples expiry dates
- 
<img width="586" alt="Screenshot 2022-05-16 at 09 30 09" src="https://user-images.githubusercontent.com/724554/168551578-75e9e9d9-2f6c-4868-b6a7-4cec56f7ca53.png">

- go to a voyage and view the select people page
- > there should be a warning message about expired documents
- > there should be an expired label next to people with an expired document
- > the checkbox should be disabled for expired people

### Notes
The way the BE currently works is 

- save people to your account
- create a voyage
- select people to add to the voyage
- > this makes a COPY of the person's details and adds them to the voyage
- (same for pleasure craft)

It does NOT connect the saved person to the voyage.
Therefore updating a saved persons details does NOT change the details saved to the voyage

This may be changed in future versions of the product.

---

**Remember to merge to feature branches not `master`.**
